### DISCUSSION TITLE
How to use json file to pass Data in components ?

### DISCUSSION THREAD

**Discussion by Shivam-apna** [websight-io/starter#86] [Q&A]
Hello!

Is there any way to pass the component data from a JSON file?

**Comment by paweljozwicki** [comment]
Hi, can you provide more details?
Is it a static file that you need just to render a component before page publication, or it can be changed later and the published page should adjust to those changes?
We are currently working on a [dynamic content example](https://github.com/websight-io/roadmap/issues/3). There we will have an example component based on dynamic data, but it's not yet ready and without a context, I don't know if it would help you.
